src/i18n/locales/sl.json

```diff
@@ -0,0 +1,105 @@
+  "sources-video-question": "Izberite željeni video vir za snemanje",
+  "sources-video-none-available": "Vaš brskalnik / naprava ne podpira zajema zaslona ali video kamere:-(",
+  "sources-video-reselect-source": "Ponovno izberi vir(e)",
+  "sources-video-no-cam-detected": "Video kamera ni bila zaznana",
+
+  "sources-audio-question": "Posnemi zvok?",
+  "sources-audio-microphone": "Mikrofon",
+  "sources-audio-without-audio": "Brez zvoka",
+
+  "sources-scenario-display-and-user": "Zaslon & video kamera",
+  "sources-scenario-display": "Zaslon",
+  "sources-scenario-user": "Video kamera",
+
+
+  "source-display-not-allowed-title": "Do vašega zaslona ni mogoče dostopati",
+  "source-display-not-allowed-text": "Poskusite znova izbrati vir. Če to ne deluje, bodisi znova naložite to stran ali ročno dovolite dostop do vašega zaslona (ponavadi preko gumba poleg naslovne vrstice brskalnika).",
+  "source-user-not-allowed-title": "Do vaše kamere ni mogoče dostopati",
+  "source-user-not-allowed-text": "Poskusite znova izbrati vir. Če to ne deluje, bodisi znova naložite to stran ali ročno dovolite dostop do vaše kamere (ponavadi preko gumba poleg naslovne vrstice brskalnika). Če to ne pomaga, poskusite zapreti druge aplikacije, ki bi lahko uporabljale to kamero: v nekaterih operacijskih sistemih lahko do kamere dostopa hkrati le ena aplikacija.",
+  "source-audio-not-allowed-title": "Do vašega mikrofona ni mogoče dostopati",
+  "source-audio-not-allowed-text": "Poskusite izbrati 'Ni zvoka' nato ponovno izberite 'Mikrofon'. Če to ne deluje, bodisi znova naložite to stran ali ročno dovolite dostop do vašega mikrofona (ponavadi preko gumba poleg naslovne vrstice brskalnika).",
+
+  "sources-display": "Zaslon",
+  "sources-user": "Video kamera",
+
+  "error-lost-video-stream": "Video snemanje se je nepričakovano končalo! Ali se je prekinila povezava s kamera ali ste po nesreči zamenjali video vir?",
+  "error-lost-audio-stream": "Snemanje zvoka se je nepričakovano končalo! Ali se je prekinila povezava s mikrofonom ali ste po nesreči zamenjali avdio vir??",
+  "error-lost-stream-end-recording": "Video ali avdio snemanje se je nepričakovano končalo in zaradi tega je bilo aktivno snemanje predčasno zaustavljeno! Ali se je prekinila povezava s video kamero ali mikrofonom? Ali ste po naključju zamenjali kamero ali avdio povezavo oziroma preklicali potrebna dovoljenja?",
+
+  "review-heading": "Ali si zadovoljen z video posnetkom?",
+  "review-error-empty-recording": "Napaka: vaš posnetek je popolnoma prazen. Ste mogoče snemanje ustavili skoraj takoj po tem, ko ste ga začeli? Poleg tega se prepričajte, da je vaš sistem dovolj zmogljiv za snemanje videa. (to morda ne velja, če druga aplikacija močno obrremenjuje vaš sistem ali pa če je vaša računalnik zelo star). V primeru  da je vaš sistem dovolj zmogljiiv za viideo snemanje, ter snemali več kot nekaj sekund in še vedno vidite to napako: Prosimo prijavite to kot napako na GitHubu.",
```

Comment:
Is "GitHubu" (at the very end of this line) a typo? If not, that's very cute :D

Reply:
Thx :) 
This is correct. In Slovenian language we usually don't  use "-" for declension of nouns.